mvpa2/generators/partition.py

```diff
@@ -491,6 +493,45 @@ def _get_partition_specs(self, uniqueattrs):
                     if ind < self.count]
 
 
+class FactorialPartitioner(Partitioner):
+    """Partitioner for two-level factorial designs
+
+    Given another partitioner on a dataset with two hierarchically organized
+    attributes, generates the combinations of all possible folds of the
+    superordinate attribute with the balanced combinations of the subordinate
+    attribute. For example, splitting familiar and unfamiliar faces while
```

Comment:
A bit more explicit, that splitting to cross validate across identities, because splitting faces is not clear... And that is under example. Use

```
Example
-----------
```